django/http/response.py

```diff
@@ -118,8 +64,8 @@ def __init__(self, content_type=None, status=None, reason=None, charset=None):
         if reason is not None:
             self.reason_phrase = reason
         elif self.reason_phrase is None:
-            self.reason_phrase = REASON_PHRASES.get(self.status_code,
-                                                    'UNKNOWN STATUS CODE')
+            self.reason_phrase = responses.get(self.status_code,
+                                               'UNKNOWN STATUS CODE')
```

Comment:
No, it should be made consistent with other codes (capfirst or title, I haven't checked).